tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -53,9 +56,19 @@ sub post_fail_hook {
     my ($self) = shift;
     reset_consoles;
     select_console('root-console');
-    if (check_var('offline_upgrade', 'yes')) {
-        $self->upload_y2logs;
-        upload_logs("/mnt/root/autoupg.xml", failok => 1);
+    if (get_var('VIRT_PRJ2_HOST_UPGRADE')) {
+        if (check_var('offline_upgrade', 'yes')) {
+            #host offline upgrade
+            $self->upload_y2logs;
+            upload_logs("/mnt/root/autoupg.xml", failok => 1);
+        }
+        else {
+            #host online upgrade
```

Comment:
I think you need:
reset_consoles;
select_console('root-ssh');
before any logs collecting.

Reply:
yep, good point! had updated to use with `select_console('root-ssh');` before any logs collecting. 
Add the latest validation run [prj2_host_upgrade_sles15sp2_to_developing_kvm](http://10.67.131.5/tests/349#downloads)